mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -341,6 +487,15 @@ class Completions(
               val include =
                 !isUninterestingSymbol(sym) &&
                   isNotLocalForwardReference(sym)
+                  && (!sym.info.typeSymbol.is(
+                    Flags.Module
+                  ) || cursorPositionCondition.isObjectValidForPos)
+                  && (cursorPositionCondition.isMethodValidForPos || !sym.is(
+                    Flags.Method
+                  ))
+                  // !sym.info.typeSymbol.is(Flags.Method) does not detect Java methods
+                  && (cursorPositionCondition.isClassOrTraitValidForPos || (!sym.isClass && !sym
+                    .is(Flags.Trait)))
```

Comment:
Same here, can you extract something to `def` to end up only with `&&` operators?